tests/Tutor.Infrastructure.Tests/TestData/EventStore/EventQueryable/EventQueryableTestCase.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text.Json;
+using Tutor.Core.BuildingBlocks.EventSourcing;
+using Tutor.Infrastructure.Database.EventStore;
+
+namespace Tutor.Infrastructure.Tests.TestData.EventStore
```

Comment:
Namespace name is not consistent with package.

Reply:
Corrected in 02acfa3104adbf25dc1aaf41773e57833916d87c.